### ISSUE TITLE
Please update for iOS 12.4 unc0ver.

### ISSUE THREAD

**Issue by nanpuhaha** [MegaCookie/SwipyFolders#13]
It doesn’t work on iPad8,3 iOS 12.4 unc0ver.

**Comment by JesseVelden** [comment]
Yup, that's true. Unfortunately it costs me really a lot of time to support this jailbreak tweak. Feel free to help though
